### DISCUSSION TITLE
服务器连接无数据

### DISCUSSION THREAD

**Discussion by hacker873** [getlantern/lantern-forum-cn#1107] [General]
服务器的问题已经反应1年了，什么时候能让我们自己选择服务器，很多自动连接的服务器我连上根本就没有数据，视频都看不了，改来改去的还是无法解决我的问题，到底能不能更改由我们自己选择好用的服务器，要是改不了退费吧，真不想用了。

**Comment by MaxMKMax** [comment]
@hacker873 

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)


当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
其实技术团队又给你们弄稳定服务器了. 不知道为什么这个问题一直在. 我再帮您反馈吧.
